platform/platform_windows.go

```diff
@@ -64,5 +129,41 @@ func GetArchInfo() (systemInfo map[string]interface{}, err error) {
 
 	systemInfo["kernel_name"] = "Windows"
 
+	// do additional work so that we don't panic() when the library's
+	// not there (like in a container)
+	si, sierr := netServerGetInfo()
+	if sierr != nil {
+		var family string
+		if (si.sv101_type & SV_TYPE_WORKSTATION) == SV_TYPE_WORKSTATION || 
+		(si.sv101_type & SV_TYPE_SERVER) == SV_TYPE_SERVER {
+			if (si.sv101_type & SV_TYPE_WORKSTATION) == SV_TYPE_WORKSTATION {
+				family = "Workstation"
+			} else if (si.sv101_type & SV_TYPE_SERVER) == SV_TYPE_SERVER {
+				family = "Server"
+			}
+			if (si.sv101_type & SV_TYPE_DOMAIN_MEMBER) == SV_TYPE_DOMAIN_MEMBER {
```

Comment:
Are we sure family will be assigned when we get here? should `family` maybe be initialized to "Unknown".  Might we end up here https://github.com/DataDog/gohai/pull/74/files#diff-cc831acf53cb0d628a7feaa8b14f4f93R154 with no assignment to `family`?

Reply:
Maybe.  But worst case it's the zero value, which is OK to assign.   The array index wasn't there before at all, so existing code won't be looking for it, and the new code will accept having "" as the value.